src/components/DateRangePicker.js

```diff
@@ -317,8 +317,7 @@ class DateRangePicker extends React.Component {
       <div className='mx-date-range-picker' style={styles.component}>
         <button
           aria-controls='calendarMenu'
-          aria-expanded={this.state.showSelectionPane}
```

Comment:
I am not too familiar with aria-expanded. May you explain why we remove it here?